src/ressources/customers/ReviewsNbField.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography, Stack } from "@mui/material";
+import CommentIcon from "@mui/icons-material/Comment";
+import {
+  useGetList,
+  useRecordContext,
+  UseRecordContextParams,
+} from "react-admin";
+import { Customer } from "./Customers";
+
+export const ReviewsNbField = (props: UseRecordContextParams<Customer>) => {
+  const record = useRecordContext<Customer>(props);
+  const { data, total, isLoading, error } = useGetList("reviews", {
+    pagination: { page: 1, perPage: 5 },
+    filter: { customer_id: record.id },
+  });
+
+  if (isLoading) {
+    return <Typography variant="body2">Loading</Typography>;
+  }
+  if (error) {
+    return <Typography variant="body2">Error</Typography>;
+  }
```

Comment:
In a real app you would probably not want to handle errors this way (you loose much information). But maybe it's not something important for the scope of this exercice.